src/test/java/net/snowflake/client/jdbc/SSOConnectionTest.java

```diff
@@ -288,6 +288,7 @@ private SFLoginInput initMockLoginInput() {
         .thenReturn(ClientAuthnDTO.AuthenticatorType.EXTERNALBROWSER.name());
     when(loginInput.getAccountName()).thenReturn("testaccount");
     when(loginInput.getUserName()).thenReturn("testuser");
+    when(loginInput.getDisableConsoleLogin()).thenReturn(true);
```

Comment:
can we add a test with value set to `false`?

Reply:
Will do once we have actual end to end test and understand how it behaves.